docs/sources/developers/plugins/legacy/review-guidelines.md

```diff
@@ -1,13 +1,10 @@
 +++
-title = "Plugin Review Guidelines"
+title = "Review guidelines"
 type = "docs"
-[menu.docs]
-name = "Plugin Review Guidelines"
-parent = "developing"
-weight = 2
+aliases = ["/docs/grafana/latest/plugins/developing/plugin-review-guidelines/"]
 +++
 
-# Plugin Review Guidelines
+# Review guidelines
```

Comment:
We need something like "Angular review guidelines" or "Legacy review guidelines" as the H1 title. This is what will show up in a documentation or Google search. I want to avoid the user getting two "Review guidelines" topics and not knowing which search result they should choose.